docs/status/template.md.jinja2

```diff
@@ -1,14 +1,5 @@
-{#
-
-This template/markdown file is auto-generated, please do not edit it directly.
-
-Please edit `branches.json` and raise a pull request, this fille will be
-automatically regenerated on merge.
-
-#}
```

Comment:
I've moved this comment to the auto-generate file

Reply:
Why? Not much point putting it there!

Reply:
But isn't that where it was meant to go? The template isn't the one that's autogenerated

Reply:
Derp.